Gimbalize/src/stage_detection.cpp

```diff
@@ -1 +1,83 @@
-//Pseudo functions of state machine 
\ No newline at end of file
+#include <Wire.h>
+#include <Adafruit_MPU6050.h>
```

Comment:
#include <MKRIMU.h>

Reply:
@namnik100 @pchou05 Should I adapt all the code for the mkr imu or keep it for the mpu and bmp?